mailbox/postgres/src/main/resources/META-INF/spring/mailbox-jpa.xml

```diff
@@ -53,7 +53,7 @@
         <constructor-arg index="1" ref="jpa-sessionMapperFactory"/>
         <constructor-arg index="2" ref="event-bus"/>
     </bean>
-    <bean id="jpa-sessionMapperFactory" class="org.apache.james.mailbox.jpa.JPAMailboxSessionMapperFactory">
+    <bean id="jpa-sessionMapperFactory" class="org.apache.james.mailbox.postgres.PostgresMailboxSessionMapperFactory">

```

Comment:
Remove this file: we will not assemble this mailbox with Spring...

Reply:
we will do it after all dependencies move to postgres, now keep it for ci pass.